lib/cylc/scheduler.py

```diff
@@ -2060,7 +2064,6 @@ def will_pause_at(self):
 
     def command_trigger_tasks(self, items):
         """Trigger tasks."""
-        print "Trigger", items
```

Comment:
This looks like a left behind debug `print` statement. The command and its arguments should already be logged by the `process_command_queue` method.